exporter/exporter.go

```diff
@@ -45,28 +50,28 @@ func (e *PaasExporter) checkForNewApps() error {
 	running := map[string]bool{}
 
 	for _, app := range apps {
-		// TODO Do we need to check they're running or does the API return all of them?
-		// need to check app.State is "STARTED"
-		running[app.Guid] = true
+		if app.State == "STARTED" {
+			running[app.Guid] = true
 
-		if _, ok := e.appNameByGuid[app.Guid]; ok {
-			if e.appNameByGuid[app.Guid] != app.Name {
-				// Name changed, stop and restart
-				e.watcherManager.DeleteWatcher(app.Guid)
-				e.createNewWatcher(app)
+			if _, ok := e.appNameByGuid[app.Guid]; ok {
+				if e.appNameByGuid[app.Guid] != app.Name {
+					// Name changed, stop and restart
+					e.deleteWatcher(app.Guid)
+					e.createNewWatcher(app)
```

Comment:
this `if` is getting quite deep but I can't think of a better way of handling this 🤔